README.md

```diff
@@ -183,7 +183,7 @@ jobs:
               repo: context.repo.repo,
               body: `**Welcome**, new contributor!
 
-                Please make sure you're read our [contributing guide](CONTRIBUTING.md) and we look forward to reviewing your Pull request shortly ✨`
+                Please make sure you've read our [contributing guide](CONTRIBUTING.md) and we look forward to reviewing your Pull request shortly ✨`
```

Comment:
Perhaps this better
```suggestion
                Please make sure you've read our [contributing guide](CONTRIBUTING.md) and we'll look forward to reviewing your Pull request shortly ✨`
```

Reply:
"look forward" is already somewhat future, you don't need another "will". You suggestion means something else: "we don't look forward to reviewing now, but we will".